### PR TITLE
Fixed docstrings for Currency Exchange

### PR DESCRIPTION
**Pull request by CosmoSt4r**
Issue #2545 

Added return type and value in docstrings for `exchange.py` and `exemplar.py` files

Additional: fixed minor typo in `tuples.py`

### CODE REVIEW

**Comment by BethanyG** [comment]
@CosmoSt4r  -- I know it is a very strong temptation to fix typos as you see them (I do that all the time!).  But for sanity/tracking sake, we need to keep most PRs grouped by concept exercise/task. 

I know it is an annoying ask, but please move the `tuples.py` change into its own PR.  Thanks!

**Comment by CosmoSt4r** [comment]
Removed typo fix in `tuples.py` from PR

File: exercises/concept/currency-exchange/.meta/exemplar.py [hunk 1]
```diff
@@ -1,71 +1,72 @@
...
-	'''
+	"""
 
 	:param budget: float - amount of money you are planning to exchange.
 	:param exchange_rate: float - unit value of the foreign currency.
 	:param spread: int - the percentage taken as an exchange fee.
 	:param denomination:  int - the value of a single bill.
-	:return:
-	'''
+	:return: unexchangeable value
```
Comment (BethanyG) [inline exercises/concept/currency-exchange/.meta/exemplar.py:67]:
```suggestion
	:return: int - unexchangeable value
```

**Review by BethanyG — APPROVED** [review]
Awesome!  Thank you for these edits and corrections.  🌟

**Pull request merged**